reana-ui/src/pages/launchOnReana/LaunchOnReana.js

```diff
@@ -44,10 +44,37 @@ export default function LaunchOnReana() {
     setLoading(true);
     client
       .launchWorkflow(Object.fromEntries(query))
-      .then(({ data: { workflow_id: workflowId } }) => {
-        dispatch(clearNotification);
-        history.push(`/details/${workflowId}`);
-      })
+      .then(
+        ({
+          data: { workflow_id: workflowId, message, validation_warnings },
+        }) => {
+          console.log(validation_warnings);
+          if (validation_warnings) {
+            let warningMessages = [];
+            // Iterate over all keys in validation_warning
+            for (const key in validation_warnings) {
+              if (key === "additionalProperties") {
+                const properties = validation_warnings[key].join(", ");
+                warningMessages.push(
+                  `Unexpected properties found in the REANA specification: ${properties}`
+                );
+              } else {
+                // For other keys, we simply display the key and its value.
+                warningMessages.push(`${key}: ${validation_warnings[key]}`);
+              }
+            }
+
+            message = `The workflow was submitted, but some warnings were issued during validation!`;
+            dispatch(triggerNotification("Workflow submitted", message));
+            warningMessages.map((warning) => {
+              dispatch(triggerNotification("Warning", warning, true));
+            });
```

Comment:
Regarding the notifications, I would only dispatch one notification instead of multiple ones, that includes both something like _The workflow was submitted, but some warnings were issued during validation._, plus _Unexpected properties found in the REANA specification: ..._. The second part can perhaps be on a separate line or we could even have an empty line between the two paragraphs, you can experiment a bit to see what looks good.

I would also use a different color/icon in this case, see for example the notification we have for workflow deletion:
![](https://github.com/reanahub/reana-ui/assets/13397702/dd59d019-e225-46a0-a304-d57a709020e1)

What do you think? Do you think it's better that we use instead the "green" notification?